packages/frontend/components/Video.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import styles from '../styles/videoBoard.module.scss'
+
+export default function Video({ video }) {
+  return (
+    <div>
+      <h3 className={styles.cardInfo}>{video.title}</h3>
+      <h4 className={styles.cardInfo}> - {video.creator}</h4>
+      <ReactPlayer className={styles.player} url={video.url} />
```

Comment:
This might help with resizing: https://www.npmjs.com/package/react-player#user-content-responsive-player

Reply:
will have a look at it, the first link is something i already tried to no avail, but as usual it is probably my fault and im beeing stupid

Reply:
after playing around for a while i still havent figured it out. Doing it the way described in the links produces some very strange results and basically breaks my stuff.  there has to be something im doing wrong